lecture_2/fh2386/fh2386.tex

```diff
@@ -0,0 +1,129 @@
+%----------------------------------------
+% Write your notes here
+%----------------------------------------
+\section{Part 1}
+\subsection{Conduct 5\% margin of error}
+\begin{documents}
```

Comment:
this should be a \begin{\itemize} to start a list instead of \begin{documents}

there's another instance of this below